### PR TITLE
ManagedIdentity - 2021-09-30-preview

### PR DESCRIPTION
**Pull request by arndt-s**
Please find new preview sdk for ManagedIdentity below.

# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

**Review by ArthurMa1978 — APPROVED** [review]

**Review by ArthurMa1978 — CHANGES_REQUESTED** [review]
Please fix the tests

**Comment by arndt-s** [comment]
@ArthurMa1978 test are fixed. I did not expect my changes would have impact on tests in other packages.

File: sdk/managedserviceidentity/Microsoft.Azure.Management.ManagedServiceIdentity/tests/Tests/IdentityTests.cs [hunk 1]
```diff
@@ -15,7 +16,7 @@ public class IdentityTests : IDisposable
     {
         private string ResourceGroupName = "SDKTest";
         private string ExplicitIdentityType = "Microsoft.ManagedIdentity/userAssignedIdentities";
-        private string CanaryCentralRegion = "centraluseuap";
+        private string USWestCentralRegion = "westcentralus";
```
Comment (ArthurMa1978) [inline sdk/managedserviceidentity/Microsoft.Azure.Management.ManagedServiceIdentity/tests/Tests/IdentityTests.cs:19]: How about name it to `TestRegion` then don't to rename again if it's changed in the future?

**Review by ArthurMa1978 — APPROVED** [review]

**Review by archerzz — CHANGES_REQUESTED** [review]
@arndt-s You need to change the https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/managedserviceidentity/Microsoft.Azure.Management.ManagedServiceIdentity/src/Properties/AssemblyInfo.cs

See the wiki, step 6: https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/managedserviceidentity/Microsoft.Azure.Management.ManagedServiceIdentity/src/Properties/AssemblyInfo.cs

File: sdk/managedserviceidentity/Microsoft.Azure.Management.ManagedServiceIdentity/tests/Tests/IdentityTests.cs (hunk 1 above)
Comment (arndt-s) [reply sdk/managedserviceidentity/Microsoft.Azure.Management.ManagedServiceIdentity/tests/Tests/IdentityTests.cs:19]: I will update this once we release this API version to public

**Review by archerzz — APPROVED** [review]

**Comment by archerzz** [comment]
@arndt-s Sorry I pasted a wrong link. It should be https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/78/Mgmt-.NET-SDK-release-process?anchor=signing-and-publishing-nuget-packges

Anyway, your change is good. Thanks.

**Pull request merged**